ofrak_core/requirements-test.txt

```diff
@@ -1,10 +1,13 @@
+angr~=9.2
```

Comment:
should be covered by `ofrak_angr` dependency - pinning it here just means we'd have to needlessly update both locations if/when updating the pin.